packages/datadog-plugin-mysql/test/integration-test/server.mjs

```diff
@@ -0,0 +1,16 @@
+import 'dd-trace/init.js'
+import mysql from 'mysql'
+
+const conn = {
+  host: 'localhost',
+  user: 'root',
+  database: 'db',
+  port: 3306
+}
+
+const connection = mysql.createConnection(conn)
+connection.connect()
+
+connection.query('SELECT NOW() AS now')
```

Comment:
Should this have an `await`? I wonder if the process could exit before the query runs...

Reply:
I'd assume not? since it was taken from the mysql example:

https://github.com/mysqljs/mysql#introduction:~:text=install%20mysqljs/mysql-,Introduction,-This%20is%20a